core/federated/RTI/rti_lib.h

```diff
@@ -23,6 +23,7 @@
 #include <strings.h>    // Defines bzero().
 #include <assert.h>
 #include <sys/wait.h>   // Defines wait() for process to change state.
+#include <signal.h>     // To trap ctrl-c and invoke a clean stop to save the trace file, if needed.
```

Comment:
Isn't this #include only needed in rti.c? If so, it should be put there.

Reply:
I guess some of the include statements above also don't belong in the header file

Reply:
> Isn't this #include only needed in rti.c? If so, it should be put there.
Moved to `rti.c`.

> I guess some of the include statements above also don't belong in the header file
Will check them.